src/NUnitFramework/framework/Attributes/TimeoutAttribute.cs

```diff
@@ -1,5 +1,5 @@
 // ***********************************************************************
-// Copyright (c) 2008 Charlie Poole, Rob Prouse
+// Copyright (c) 2018 Charlie Poole, Rob Prouse
```

Comment:
I've been doing ranges, but I don't remember any discussion on this.

Reply:
Currently, ~ 9% of our headers use range, and most of them was inserted in 2015/2016. In this PR this is now a range.

Reply:
That makes sense. Only those files that had changed significantly since we started doing it would have ranges. I've been adding or advancing the second year in the range when I make significant changes to a file. This is the first time I've seen anyone wipe out the original date.

Reply:
A later commit in this PR reverted the code, so it is now a range, see e.g. https://github.com/nunit/nunit/pull/3027/commits/24d6d04b61e53527e748c1b2feced72e389655ba#diff-d84772ba8bf03c8aa4d1299488b49f7dR2